controllers/controllers.js

```diff
@@ -0,0 +1,7 @@
+const { selectTopics } = require('../models/models.js')
+
+exports.getTopics = (req, res) => {
+    selectTopics().then((topics) => {
+        res.send(topics)
```

Comment:
We should be returning an object with the key of topics containing an array of all the topics, rather than just an array of topics. It's good practise and will help us extend our API later.